src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(String color, double side) {
```

Comment:
Consider using separate variables for the constructor parameters for better readability instead of passing expressions directly.